packages/apollo-engine-reporting/src/treeBuilder.ts

```diff
@@ -15,6 +15,7 @@ export class EngineReportingTreeBuilder {
     [rootResponsePath, this.rootNode],
   ]);
   private rewriteError?: (err: GraphQLError) => GraphQLError | null;
+  private consolePrefix = '[apollo-engine-reporting]';
```

Comment:
how is this helpful in errors? isn't the stack trace good enough?

Reply:
I'm a fan of this as it's a clearer indication that something is seriously fucked and it's not part of the implementing application.

Reply:
ok, but in that case maybe the message should literally say it's an internal bug in the library?